templates/register/outside/reset_password_complete.html

```diff
@@ -0,0 +1,32 @@
+{% load staticfiles %}{% load crispy_forms_tags %}
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+		<meta charset="utf-8">
+		{% include "header.html" %}{% block head-extra %}{% endblock %}
+		<link href="{% static 'complete_registration.css' %}" rel="stylesheet" media="screen">
+		<link href="{% static 'login_style_additions.css' %}" rel="stylesheet" media="screen">
+	</head>
+
+	<body>
+		<div class="container text-center">
+			<div class="headline">
+				<svg class="armada-logo" viewBox="305 16 320 150" width="555px" height="250px" version="1.1">
+					<image xlink:href="{% static 'images/armada_logo_text_left_green.svg' %}" alt="Armada" height="250px" width="780px"/>
+				</svg>
+				<h2>Reset password complete</h2>
+				<p class="less-important-text">Your password has been set. You may go ahead and log in now.</p>
+				<br>
+			</div>
+			<div class="text-center form-container">
+				<a href="{% url 'anmalan:login' %}" class="btn-signin btn-green">Sign in</a>
+				<div>
+					<svg viewBox="50 90 200 200" width="100px" height="180px" version="1.1">
+						<image xlink:href="{% static 'images/armada_logo_text_left_green.svg' %}" alt="Armada" height="300px" width="700px"/>
+					</svg>
+				</div>
+			</div>
+		</div>
+	</body>
+</html>
```

Comment:
This was created from the template `reset_password_done.html` which the text from default django page and a button for easy access to login screen.